### ISSUE TITLE
Connection could not be established with host mail.domain.com [Connection timed out #110] 

### ISSUE THREAD

**Issue by distresscode** [freescout-help-desk/freescout#4441]
Hi all,

I hope you can spare a minute during the holidays season and help me troubleshoot this issue that started suddenly on its own. 

TLDR; I cannot send or fetch emails in Freescout for all my mailboxes. The mail settings test in FS sends and is being received just fine. Only mailboxes refuse to work. 

![0ced0ad2-42b4-4d50-821b-0f82080dcb9a-image](https://github.com/user-attachments/assets/8c7d87e1-d03a-4c71-bd5f-4422f8e67bc7)

FreeScout 1.8.160 
Cloudron package 1.11.19

Log errors are (fetch and send logs, respectively):

```
Error: connection failed - stream_socket_client(): Unable to connect to ssl://mail.domain.com:993 (Connection timed out); File: /app/code/overrides/webklex/php-imap/src/Connection/Protocols/ImapProtocol.php (94)

 Connection could not be established with host mail.domain.com [Connection timed out #110]; File: /app/code/overrides/swiftmailer/swiftmailer/lib/classes/Swift/Transport/StreamBuffer.php (271) 
```
` Manage > Mailboxes > Connection Settings > Sending Emails > Send Test To` & ` Manage > Mailboxes > Connection Settings > Fetch Emails > Send Test To`, result in error saying make sure ports 587 and 993 are open. I checked using an online website by entering server IP and port 587 and then 993 and can confirm both ports are indeed, OPEN!

Here are some troubleshooting steps I've taken:

* Manage > System > Status: It's all OK and queue:work 	Running
* Manage > System > Tools > Clear Cache
* Restarted app manually
* App updated overnight to package 1.11.19 (issue persisted)
* Checked ports 587 and 993 using online website and confirmed they are OPEN
* Checked logs as mentioned in FS Troubleshooting page and posted results above
* Created a new app password (mail) to use with one of my mailboxes (not the issue as my other FS mailboxes aren't working too)
* Can confirm Cloudron mailserver is working fine as I can send/receive mail in Roundcube to all mailboxes including my Freescout mailboxes. I can also check/send mail from iOS mail app.
* Freescout test mail settings sent and received the email just fine so the mail settings (SMTP Mailgun) is working in the app. It just refuses to send/fetch at the mailbox level.

Your 2c are highly appreciated! Thanks and Happy Holidays to you all.

**Comment by distresscode** [comment]
It turned out to be a docker issue - still investigating the bug tho

**Comment by dennemark** [comment]
what was the docker issue, can you elaborate?
